lib/ansible_tower_client/inventory.rb

```diff
@@ -4,6 +4,16 @@ def self.endpoint
       "inventories".freeze
     end
 
+    def inventory_sources
+      api.inventories.find_all_by_url(related['inventory_sources'])
```

Comment:
@bdunne - Per your comment above - I looked into this - but realized for `root_groups` we're doing the same lookup - should we rename `find_all_by_url`? I like the utility it provides with only a url as an argument. 

Plus `find_all_by_url` is already calling `collection_for` but without all the nested arguments.